### PR TITLE
Zh tn bug 240712

### PR DESCRIPTION
**Pull request by BuyuanCui**
# What does this PR do ?

Fixing NvBug for Money grammar. Treating capital "L" as currency.

# Before your PR is "Ready for review"
**Pre checks**:
- [*] Have you signed your commits? Use ``git commit -s`` to sign.
- [*] Do all unittests finish successfully before sending PR?
   1) ``pytest`` or (if your machine does not have GPU) ``pytest --cpu`` from the root folder (given you marked your test cases accordingly `@pytest.mark.run_only_on('CPU')`).
   2) Sparrowhawk tests ``bash tools/text_processing_deployment/export_grammars.sh --MODE=test ...``
- [NA] If you are adding a new feature: Have you added test cases for both `pytest` and Sparrowhawk [here](tests/nemo_text_processing).
- [*] Have you added ``__init__.py`` for every folder and subfolder, including `data` folder which has .TSV files?
- [*] Have you followed codeQL results and removed unused variables and imports (report is at the bottom of the PR in github review box) ?
- [*] Have you added the correct license header `Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` to all newly added Python files?
- [*] If you copied [nemo_text_processing/text_normalization/en/graph_utils.py](nemo_text_processing/text_normalization/en/graph_utils.py) your header's second line should be `Copyright 2015 and onwards Google, Inc.`. See an example [here](https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/graph_utils.py#L2).
- [*] Remove import guards (`try import: ... except: ...`) if not already done.
- [NA] If you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
- [*] Have you added your language support to [tools/text_processing_deployment/pynini_export.py](tools/text_processing_deployment/pynini_export.py).
  

**PR Type**:
- [ ] New Feature
- [*] Bugfix
- [ ] Documentation
- [ ] Test

If you haven't finished some of the above items you can still open "Draft" PR.

### CODE REVIEW

File: nemo_text_processing/text_normalization/zh/taggers/money.py [hunk 1]
```diff
@@ -117,24 +116,24 @@
...
             + pynutil.insert(" ")
             + pynini.closure(graph_decimal, 1)
-            + pynini.closure(pynutil.insert(" ") + pynutil.insert('quantity: \"') + suffix + pynutil.insert('\"'))
+            + pynini.closure(
+                (pynutil.insert(" ") + pynutil.insert('quantity: \"') + suffix + pynutil.insert('\"')), 0, 1
+            )
         )
 
-        graph = (
+        final_graph = (
```
[the remark this answers is not in the document]
Comment (ekmb) [reply nemo_text_processing/text_normalization/zh/taggers/money.py:131]: @BuyuanCui could you address this one?

[the remark this answers is not in the document]
Comment (zoobereq) [reply nemo_text_processing/text_normalization/zh/taggers/money.py:131]: Maybe rename `final_graph` as `graph` and pass it to `self.add_tokens`?

**Review by zoobereq — CHANGES_REQUESTED** [review]
Not much to add beyond what @ekmb pointed out

**Comment by zoobereq** [comment]
It looks like jenkins is complaining. Unfortunately, I don't have access to take a look at what may be the issue. @ekmb maybe you could take a peek?

Did `Sparrowhawk` pass locally for this MR?

**Comment by BuyuanCui** [comment]
> It looks like jenkins is complaining. Unfortunately, I don't have access to take a look at what may be the issue. @ekmb maybe you could take a peek?
> 
> Did `Sparrowhawk` pass locally for this MR?

I might be that it is complaining about the conflict on the date. I resolved the conflict and pushed it. Hopefully it will pass.

**Review by tbartley94 — APPROVED** [review]
LGTM

**Comment by tbartley94** [comment]
@BuyuanCui what's the status on this branch?

**Comment by BuyuanCui** [comment]
> @BuyuanCui what's the status on this branch?

Hi, I see the EN fixes are applied, just resolved the conflicts. Hoping to see all tests to pass.

**Review by zoobereq — APPROVED** [review]
Looks good

**Pull request merged**